tx-source/site_core.php

```diff
@@ -1365,6 +1388,7 @@
 $Definition['Your email has been successfully confirmed.'] = 'Your email has been successfully confirmed.';
 $Definition['Your invitation has been sent.'] = 'Your invitation has been sent.';
 $Definition['You haven\'t joined any groups yet.'] = 'You haven\'t joined any groups yet.';
+$Definition['Your login credentials are:'] = 'Your login credentials are:';
```

Comment:
Concatenating login credentials here is not correct localization. You need a %s or formatString() compatible string.